samples/prot_test/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _prot_test:
+
+Protection tests
+#################################
+
+Overview
+********
+This test provides a set of shell commands to test
```

Comment:
Your commit message asked whether this would be better placed in "tests" rather than "samples" -- I think it would be better in "tests" since this isn't really an example of how to do something useful.